app/tx/[signature]/page-client.tsx

```diff
@@ -229,7 +263,13 @@ function StatusCard({ signature, autoRefresh }: SignatureProps & AutoRefreshProp
                         <td>Error</td>
                         <td className="text-lg-end">
                             <h3 className="mb-0">
-                                <span className={`badge bg-${statusClass}-soft`}>{errorReason}</span>
+                                {errorLink ? (
+                                    <Link href={errorLink}>
+                                        <span className={`badge bg-${statusClass}-soft`}>{errorReason}</span>
+                                    </Link>
+                                ) : (
+                                    <span className={`badge bg-${statusClass}-soft`}>{errorReason}</span>
+                                )}
```

Comment:
This actually works surprisingly well. No UI indicator mentioning it's a link, but still great.